src/python/ensembl/io/genomio/genbank/extract_data.py

```diff
@@ -399,22 +456,30 @@ def _write_seq_region_json(self):
                     "url": "https://www.ncbi.nlm.nih.gov/genbank",
                 },
             }
-            if not seq_obj["added_sequence"]["assembly_provider"]["name"]:
+            if not seq_obj["added_sequence"]["assembly_provider"]:
```

Comment:
Not sure what we're doing here: we define `seq_obj["added_sequence"]` just above, so this check will never fail?

Reply:
yes exactly, maybe I will just remove it and add this to the pytest